artifacts/definitions.py

```diff
@@ -22,24 +25,32 @@
     'Containerd': 'Containerd artifacts',
     'Docker': 'Docker artifacts.',
     'Execution': 'Contain execution events.',
-    'ExternalAccount': (
+    'ExternalAccount': '<Deprecated>', # TODO: to be deleted
+    'External Account': (
         'Information about any user accounts e.g. username, '
         'account ID, etc.'),
     'External Media': 'Contain external media data or events e.g. USB drives.',
+    'File System': 'Artifacts related to the file system e.g. MFT.',
     'Hadoop': 'Hadoop artifacts.',
     'IM': 'Instant Messaging / Chat applications artifacts.',
+    'Interactive': 'Artifacts created by interactive (e.g. RDP) user activity.',
     'iOS': 'Artifacts related to iOS devices connected to the system.',
     'History Files': 'History files artifacts e.g. .bash_history.',
-    'KnowledgeBase': 'Artifacts used in knowledge base generation.',
+    'KnowledgeBase': '<Deprecated>', # TODO: to be deleted
+    'Knowledge Base': 'Artifacts used in knowledge base generation.',
     'Kubernetes': 'Kubernetes artifacts',
     'Logs': 'Contain log files.',
     'Mail': 'Mail client applications artifacts.',
     'Memory': 'Artifacts retrieved from memory.',
     'Network': 'Describe networking state.',
+    'Persistence' : 'Persistence mechanisms e.g. the Startup folder.',
```

Comment:
This label likely looses its value since there are many persistence mechanisms

Reply:
Also there are already groups for this purpose such WindowsPersistenceMechanisms and WindowsPersistenceRegistryKeys. Why is an additional label needed? what is the use case?

Reply:
The use case is to iterate over all artifact definitions and pick up those that have the label Persistence. What's the downside of having the label and adding it to all artifact definitions that relate to maintaining persistence?

Reply:
> What's the downside of having the label and adding it to all artifact definitions that relate to maintaining persistence?

note that this label is applicable to a lot of definitions. If it is added to most of the artifact definitions the label, the label looses its value as way to easily filter on persistence related artifacts.